internal/errors/code.go

```diff
@@ -34,6 +34,7 @@ const (
 	SessionNotFound       Code = 110 // SessionNotFound represents that the session was not found
 	InvalidSessionState   Code = 111 // InvalidSessionState represents that the session was in an invalid state
 	TokenMismatch         Code = 112 // TokenMismatch represents that there was a token mismatch
+	TooShort              Code = 113 // TooShort represents an error that means the provided input is not meeting minimum length requirements
```

Comment:
story of my life in an error code.

Reply:
"you must be this tall to run this program"